markdown-scheme_v3.Rmd

```diff
@@ -0,0 +1,393 @@
+        \begin{enumerate}
+          \item Name
+          \item Data set ID: (empty)
+          \item Note: (empty)
+          \item Access policy:
+            \begin{itemize}
+              \item Published elsewhere: Note access/license
+              \item Published with package: Authors must confirm there is no confidentiality.
+            \end{itemize}
+          \item License (empty)
+          \item License URL (empty)
+          \item Data URL
+        \end{enumerate}
+\end{enumerate}
+
+## Methods, software and scripts
+Fields that are not listed can be discarded.
+\begin{enumerate}
+  \item Methods: Econometric methods used. \textcolor{red}{This field will learn from past entries.} Possibly add notes (more details about the context)
+  \item Software: Stata 17.0, R, etc
+  \item Scripts:
+      \begin{enumerate}
+          \item File name (WP+PP): Name of the zip file. Naming convention: PrimaryID\_prg\_v01.zip
+          \item Zip package (WP+PP): Same as file name\footnote{File name and zip package are identical, because we do not list files separately, for which the field file name is there.}
+          \item Title: 
+            \begin{lstlisting}[breaklines]
+              Reproducibility package (data and code) for Title
+            \end{lstlisting}
+            Title may be shortened if adequate.
+          \item Authors: WP+PP: empty\footnote{According to descr: A list of authors and coauthors of the script} \textcolor{red}{If authors of the paper and the code differ and authors want to indicate this, add here list of authors for the code!}
+          \item Date: Year-Month (usually the completion/version date)
+          \item Software: Stata
+          \item Description: empty
+          \item Methods: \textcolor{red}{This is methods used in the script}. Usually empty
+          \item Dependencies: Here enter all packages necessary:\footnote{It may be preferable to enter this in Software, libraries: \textcolor{red}{Error: Must be an array}. Does not accept more than one entry, and also not a string. Behavior of the editor his random here}
+            \begin{lstlisting}
+              coefplot, distinct, lincomeest, lprobust.ado
+            \end{lstlisting}
+          \item Instructions: Write: "See readme in reproducibility package"
+          \item Repository: ICPSR, Harvard Dataverse, Journal webpage etc. For ICPSR
+                the correct description is
+                \begin{lstlisting}
+                  AEA Data and Code Repository (ICPSR)
+                  Reproducible Research Repository (World Bank)
+                \end{lstlisting}
+          \item Notes: \textcolor{red}{Here who verified the package}. 
```

Comment:
I think we may want a specific field for reproducibility verification, we should discuss with Olivier about adding this. Notes field is a fine workaround for now.